platform/features/imagery-timeline/res/templates/timeline.html

```diff
@@ -0,0 +1,9 @@
+
+<div class="l-image-thumbs-wrapper"ng-controller="ImageryTimelineController">
+	<div class="l-image-thumb-item" ng-repeat="image in images">
+		<img class="l-thumb"
+			src={{image['url']}}>
+			<div class="l-date">{{image['utc']}}</div>
+			<!-- <div class="l-time">16:03:0{{$index}}</div> -->
```

Comment:
I know it's a WIP so okay, but when finalized there should be no commented out code.